packages/daemon/src/generator/tvmGeneratorParent.ts

```diff
@@ -0,0 +1,19 @@
+import { fork } from 'child_process';
```

Comment:
@WenheLI As we have discussed, we should name this file as "generator/tvm.ts", and rename the `tvmGenerator` to another directory, or `generator/tvm.cli.ts`, so that we could get its relationship.